src/libs/transformations/assignMeTo.js

```diff
@@ -0,0 +1,26 @@
+import pathnameToRoute from "../helpers/pathnameToRoute";
+import ROUTES from "../ROUTES";
+
+export default () => {
+  const route = pathnameToRoute(location.pathname);
+  let x, y; // eslint-disable-line one-var
+  if (route === ROUTES.ISSUES || route === ROUTES.MRS) {
+    document.onmousemove = e => {
+      x = e.pageX;
+      y = e.pageY;
+    };
+  }
+  document.body.onkeydown = function(e) {
+    if (e.keyCode === 32 || (e.keyCode === 65 && e.altKey)) {
```

Comment:
This will handle all "space" keydowns... e.g. even when writing text in textarea 

Try using Mousetrap (already present in Gitlab frontend code), viz https://github.com/kiwicom/refined-gitlab/blob/2d16647ce88e3647e48c6eb1a93e06559e541f90/src/agent.js#L30